plugins/catalog-node/api-report-alpha.md

```diff
@@ -88,7 +88,7 @@ export interface CatalogProcessingExtensionPoint {
 export const catalogProcessingExtensionPoint: ExtensionPoint<CatalogProcessingExtensionPoint>;
 
 // @alpha
-export const catalogServiceRef: ServiceRef<CatalogApi, 'plugin'>;
+export const catalogServiceRef: ServiceRef<CatalogApi, 'plugin', true>;
```

Comment:
Hmm, maybe it wouldn't be bad to have this be an enum instead for readability, like `'singleton' | 'collected'`. We'd need to find yet another name for the option in that case though 😅

Reply:
What about `'single' | 'multiple'`?

Reply:
We ended up using 'singleton' and 'multiton'